web-app/server/fabric/wallet/admin.id

```diff
@@ -0,0 +1 @@
+{"credentials":{"certificate":"-----BEGIN CERTIFICATE-----\nMIIB8jCCAZmgAwIBAgIUUXH7QI32WoRp5WtPx3X4P+kYpCkwCgYIKoZIzj0EAwIw\ncDELMAkGA1UEBhMCVVMxFzAVBgNVBAgTDk5vcnRoIENhcm9saW5hMQ8wDQYDVQQH\nEwZEdXJoYW0xGTAXBgNVBAoTEG9yZzEuZXhhbXBsZS5jb20xHDAaBgNVBAMTE2Nh\nLm9yZzEuZXhhbXBsZS5jb20wHhcNMjEwNjI4MjMzMjAwWhcNMjIwNjI4MjM0MDAw\nWjAhMQ8wDQYDVQQLEwZjbGllbnQxDjAMBgNVBAMTBWFkbWluMFkwEwYHKoZIzj0C\nAQYIKoZIzj0DAQcDQgAEEsCQhB71k9V1O6HJ772rHkCyQoTiEwj5HHEDAZOJO/6U\nbSDapU6y9rZaIHkW1o7WkeiA5B4IQswndbdsBYsYKqNgMF4wDgYDVR0PAQH/BAQD\nAgeAMAwGA1UdEwEB/wQCMAAwHQYDVR0OBBYEFClHG3N8muchPrLEccCBThlZz/0Y\nMB8GA1UdIwQYMBaAFMXvVBoLlt+9bOFgEwAwpep3VgzTMAoGCCqGSM49BAMCA0cA\nMEQCIHspvaEhOvnL0Etsk8CDHIzGS87hInE97OgYH3Jm2pVyAiBSj481m7AqgpwN\nzdHDtK54owZUdEShTyYPT+eCGAwwoQ==\n-----END CERTIFICATE-----\n","privateKey":"-----BEGIN PRIVATE KEY-----\r\nMIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgGCkYRsMMBtLQBa9w\r\nz0Slg26BHOPMPa7LE5NUS8WX5NahRANCAAQSwJCEHvWT1XU7ocnvvaseQLJChOIT\r\nCPkccQMBk4k7/pRtINqlTrL2tlogeRbWjtaR6IDkHghCzCd1t2wFixgq\r\n-----END PRIVATE KEY-----\r\n"},"mspId":"Org1MSP","type":"X.509","version":1}
```

Comment:
Maybe we should add the `wallet` folder in the .gitignore. Let's discuss this during the next call.